src/m4a.c

```diff
@@ -1525,6 +1525,10 @@ void ply_xwave(struct MusicPlayerInfo *mplayInfo, struct MusicPlayerTrack *track
 {
     u32 wav;
 
+#ifdef UBFIX
```

Comment:
There actually isn't any undefined behavior with these of these zero assignments, due to the bit twiddling that occurs.

Reply:
I don’t see any such thing. I see var being &=, which means the value is reassigned based on the previous value.

Reply:
Yes, but you're ignoring what the result of the operations is.  All four groups of 8-bits in the 32-bit value are written individually such that all bits are accounted for, and therefore no undefined behavior.  This is true for `ply_xcmd_0C` because it operates on a 16-bit value there.

Reply:
That being said, there might still be an argument to explicitly set the 0, like you've done.  For example, on a 64-bit system, this would be undefined beahvior.

Reply:
I think it’s best to have it here just in case, just so the compiler at least is aware, plus safer code isn’t a bad thing. Otherwise I will just remove them.